### PR TITLE
Address utils

### PR DESCRIPTION
**Pull request by bluesign**
adds AddressUtils.currentNetwork and AddressUtils.parseAddress

```
     log(AddressUtils.currentNetwork())
     log(AddressUtils.parseAddress("0x1"))
     log(AddressUtils.parseAddress(1))
     log(AddressUtils.parseAddress(0x1))
     log(AddressUtils.parseAddress("0xf8d6e0586b0a20c7"))
     log(AddressUtils.parseAddress(AddressUtils.getType()))
```

### CODE REVIEW

File: cadence/contracts/AddressUtils.cdc [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import StringUtils from "./StringUtils.cdc"
+
+pub contract AddressUtils {
+
+    priv fun sansPrefix(_ input: String): String{
```
Comment (bjartek) [inline cadence/contracts/AddressUtils.cdc:5]: /
```suggestion
    priv fun withoutPrefix(_ input: String): String{
```

File: cadence/contracts/AddressUtils.cdc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            stringValue = (input as! Int).toString()
+        }
+        else if input.getType().isSubtype(of: Type<Address>()){
+            stringValue = (input as! Address).toString()
+        }
+        else if input.getType().isSubtype(of: Type<Type>()){
+            stringValue =  StringUtils.split((input as! Type).identifier, ".")[1]
+        }
+
+        var address=self.sansPrefix(stringValue)
```
Comment (bjartek) [inline cadence/contracts/AddressUtils.cdc:36]:
```suggestion
        var address=self.withoutPrefix(stringValue)
```

**Review by bjartek — APPROVED** [review]
Should we add some tests maybe?

**Comment by bluesign** [comment]
yeah we should move testing to overflow also. JS tests I am super noob ( or cadence testing )

**Comment by austinkline** [comment]
I'm good with moving to overflow, no strong preferences there. Would you rather I wait for tests or merge/deploy this now?

**Review by austinkline — APPROVED** [review]

**Comment by bjartek** [comment]
Just merge

**Pull request merged**